model/issue.go

```diff
@@ -3,11 +3,15 @@
 
 package model
 
+import (
+	"github.com/mattermost/mattermost-mattermod/types"
+)
+
 type Issue struct {
 	RepoOwner string
 	RepoName  string
 	Number    int
 	Username  string
 	State     string
-	Labels    []string
+	Labels    types.JSONText
```

Comment:
Perhaps `StringArray` is a better name for this? To keep the parity with `[]string`.

Let's also keep this in the model package itself.

Reply:
👍  Will update the name and location of the type